features/step_definitions/moderation_steps.rb

```diff
@@ -3,6 +3,11 @@
   visit edit_admin_petition_path(@petition)
 end
 
+When /^I click at the sponsored petition with title: "([^"]*)", that has action: "([^"]*)" and description: "([^"]*)"$/ do |title, action, description|
+  @sponsored_petition = FactoryGirl.create(:sponsored_petition, title: title, action: action, description: description)
+  visit edit_admin_petition_path(@sponsored_petition)
+end
```

Comment:
This step merges data creation and user action.  It's not a major sin or anything, but we should be wary of it.

However, I think my main concern is the name "I click at..." which suggests a user looking at something on screen and clicking on it, when in fact we just point their browser directly at the edit page for them.  Maybe just change the title to "I visit ...." ?